src/test/java/org/alfresco/repo/security/authority/AuthorityServiceTest.java

```diff
@@ -252,6 +252,20 @@ public void testMNT10533() throws Exception
         }
     }
 
+    public void testCreateAuthorityWithIllegalCharacters() throws Exception
+    {
+        char[] illegalCharacters = {'/', '\\', '\n', '\r', '"'};
+        for (char illegalCharacter : illegalCharacters)
+        {
+            String GROUP_NAME = "testGroupNameWith" + illegalCharacter;
+            try {

```

Comment:
Please check your code style. Braces are in wrong places.

Reply:
Thanks for the spotting it, just linted it.